apps/resources/scala3.json

```
@@ -0,0 +1,12 @@
+{
+  "mainClass": "dotty.tools.MainGenericRunner",
+  "repositories": [
+    "central"
+  ],
+  "dependencies": [
+    "org.scala-lang:scala3-compiler_3:3.1.0-RC1"
```

We don't have to hard-code the version here. We can use `latest.release` for now (that will also pick nightlies…), and switch to `latest.stable` as soon as a non-nightly / non-RC version works.

(I just realized this also applies to other `scala3-*` apps, https://github.com/coursier/apps/pull/109.)

```suggestion
    "org.scala-lang:scala3-compiler_3:latest.release"
```